tinyms/serving/server/server.py

```diff
@@ -164,7 +177,7 @@ def run_flask(host='127.0.0.1', port=5000):
     app.run(host=host, port=port)
 
 
-def start_server(host='127.0.0.1', port=5000):
+def start_server(host='127.0.0.1', port=5000, servable_path=None, ckpt_path=None):
```

Comment:
I'm not sure if `ckpt_path` and `servable_path` is required here, because this sever could contain multiple servables at the same time. Instead, I suggest to pass a `serving_path` (default is `/etc/tinyms/serving`) in this function.

Reply:
Changing the default directory is indeed the way to go, I'll go back and update it tonight